src/scroll-timeline-css-parser.js

```diff
@@ -84,9 +84,17 @@ export class StyleParser {
     }
 
     // If this sheet has no srcURL (like from a <style> tag), we are done.
-    // TODO: Otherwise, we have to find `url()` functions and resolve
-    // relative and path-absolute URLs to absolute URLs.
-    return p.sheetSrc;
+    // Otherwise, we have to find `url()` functions and resolve
+    // relative URLs to absolute URLs.
+    if (!srcUrl) {
+      return p.sheetSrc;
+    }
+
+    const srcUrlDir = srcUrl.lastIndexOf('/') > location.origin.length
+        ? srcUrl.substring(0, srcUrl.lastIndexOf('/'))
+        : location.origin;
```

Comment:
Why use location.origin instead of using the same srcUrl.substring expression here?

Reply:
Because I thought it could possibly resolve to just `https://`

Reply:
If we load the css from `https://subdomain.example.com` exactly (and the stylesheet is the response)

Reply:
Ah that makes sense, but I know there have been requests to support loading from other origins when CORS is enabled - see #248 - so it'd be great to still use the srcUrl.

Reply:
Makes sense. I now use the origin of srcUrl.